README.rst

```diff
@@ -61,12 +63,57 @@ Installation
             ansible_connection: local
           hosts:
             gcp-staging:
-              k8s_kube_context: <name of context from ~/.kube/config>
+              k8s_auth_host: <https://....Cluster API endpoint URL.....>
               k8s_domain_names:
               - www.example.com
 
 See ``defaults/main.yml`` for all the variables that can be overridden.
 
+The ``k8s_auth_host`` variable is absolutely required to be set. This is the API
+endpoint URL of the cluster to use. Here are some examples so you can see what
+it might look like:
+
+AWS: ``https://74406E3AD450E7845D0EF653E7C6F020.gr7.us-west-2.eks.amazonaws.com``
```

Comment:
@copelco what would this URL look like for a cluster in Azure?

Reply:
Something like this:
```suggestion
AKS: ``https://ratom-staging-dns-ba5d6fd2.hcp.eastus.azmk8s.io:443``

AWS: ``https://74406E3AD450E7845D0EF653E7C6F020.gr7.us-west-2.eks.amazonaws.com``
```